doc/development/plugins.rst

```diff
@@ -114,7 +115,7 @@ then finite shots exist. If ``shots`` is falsy, then an analytic execution shoul
 Preprocessing
 -------------
 
-The :meth:`~.devices.Device.preprocess` method has two main responsibilities:
+There are two components of preprocessing circuits for device execution:
```

Comment:
Mind updating Line 123 below:

```
These two tasks can be extracted into private methods or helper functions if that improves source
code organization.
```

```
These two tasks equate to ``preprocess_transforms`` and ``setup_execution_config``.
```

They don't need to be private anymore.

Reply:
True...